segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaConnectionListener.java

```diff
@@ -277,4 +277,4 @@ public void close() {
             tlsCertFileModificationMonitor.stopMonitoring();
         }
     }
-}
+}
```

Comment:
This change is unrelated to this PR, please undo it.

Reply:
Done.